data/coalition outfits.txt

```diff
@@ -30,6 +34,8 @@ outfit "Large Collector Module"
 
 outfit "Small Reactor Module"
 	category "Power"
+	licenses
+		Coalition
```

Comment:
If I remember correctly this is one of the Heliarch outfits. If it is then it should be behind a Heliarch license.